teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -2,127 +2,130 @@
-Catalog, but I don't want to do much more work than that. I'm willing to deal with bugs reported and pull requests
+As a casual Contributor to the Tekton Catalog, I have created a `Task` that works for me, and I'd like to submit it to
+the Catalog, but I don't want to do much more work than that. I'm willing to deal with bugs reported and pull requests
 opened for it, but I don't want to have to bother submitting tests with it.
 
 #### Dedicated Contributor
 
-As a dedicated contributor to the Tekton Catalog, I have created a Task and I want to make sure it continues to work
-over time. I'm willing to put in the time to create a test but I want to understand exactly how to create that test
-without having to track down a maintainer to help me. Moreover, I want to sign the Task to mark it as trusted.
+As a dedicated Contributor to the Tekton Catalog, I have created a `Task` and I want to make sure it continues to work
+over time. I'm willing to put in the time to create a test, but I want to understand exactly how to create that test
+without having to track down a Maintainer to help me. Moreover, I want to sign the `Task` to mark it as trusted.
 
 #### Tekton Maintainer
 
-As a maintainer of a Tekton project, I have a Task which I would like to be an official part of Tekton and I would
-like other Tekton maintainers to help maintain over time. In addition to automated testing for the Task, I want the
-image used in the Task to be regularly scanned for common vulnerabilities and exposures so that we ensure *official*
-Tasks are secure. I also want this official Task to demonstrate best and secure practices that users can confidently
-imitate when authoring their own Tasks. Even more, I want to dogfood features and components in Tekton to gather
+As a Maintainer of a Tekton project, I have a `Task` which I would like to be an verified part of Tekton and I would
+like other Tekton Maintainers to help maintain over time. In addition to automated testing for the `Task`, I want the
+image used in the `Task` to be regularly scanned for common vulnerabilities and exposures so that we ensure verified
+`Tasks` are secure. I also want this verified `Task` to demonstrate best and secure practices that users can use as a
+sample when authoring their own `Tasks`. Even more, I want to dogfood features and components in Tekton to gather
 feedback and iterate quickly.
 
 ### Goals
 
 #### Ownership and Maintenance
 
-Every resource in the *Tekton Catalog* needs to have Owners to maintain them. The Ownership needs to be distributed among
-community members and Tekton maintainers to ensure that the workload is manageable and sustainable.
+Every resource in the [`tektoncd-catalog`](https://github.com/tektoncd-catalog) GitHub organization needs to have Owners to maintain them. The Ownership needs to be distributed among community members and Tekton Maintainers to ensure that the workload is manageable and sustainable.
```

Comment:
should we also define how one can add their resource in https://github.com/tektoncd-catalog repository if they want to?
cc @jerop @vdemeester

Reply:
That's one reason to go to ArtifactHub, it's "easy" to get any task "indexed" (by ArtifactHub) without putting the burden on the tekton-catalog maintainers.